.gitignore

```diff
@@ -31,3 +31,4 @@ bin/
 _book
 node_modules
 data/example/exampleObj_file1.txt
+example/mapreduce-example/mapreduce-example-runner/data
```

Comment:
These should be deleted by the test code and example code. I'll put in an issue to clean these up.